src/components/tippy.js

```diff
@@ -1,5 +1,14 @@
 import tippy from "tippy.js";
 
 export default (node, props) => {
-  tippy(node, props);
+  let instance = props && tippy(node, props);
+  return {
+    update(newProps) {
+      if (instance) instance.destroy();
+      if (newProps) instance = tippy(node, newProps);
+    },
+    destroy() {
+      if (instance) instance.destroy();
+    },
+  };
```

Comment:
I wasn't able to understand why this change was necessary. According to the PR description, is it to "fix some bugs in our usage of tippy.js which made descriptions not work with the itemlist"?

Reply:
In the metric list group, labels may be updated or destroyed as the user paginates through the content. This could result in stale content appearing for a label.

Reply:
ah I see what you mean. Thanks for the explanation!